src/components/abstractComponent.ts

```diff
@@ -502,78 +408,29 @@ export abstract class AbstractComponent<D> implements IComponent<D> {
     this._parent = parent;
     return this;
   }
-  /**
-   * @returns {Bounds} for the component in pixel space, where the topLeft
-   * represents the component's minimum x and y values and the bottomRight represents
-   * the component's maximum x and y values.
-   */
-  public bounds() {
```

Comment:
Sorry I guess my original comment wasn't clear; we can still expose these as top-level Component methods if they make sense, but we just don't need to expose them in the interface directly since they have an implementation already.